pkg/cliplugins/workspace/plugin/kubeconfig_test.go

```diff
@@ -1161,3 +1343,56 @@ func (c errorDiscoveryClient) ServerGroups() (*metav1.APIGroupList, error) {
 	}
 	return c.DiscoveryInterface.ServerGroups()
 }
+
+// TODO(nrb): this is a modified copy from apibinding reconcile. Unify these into a reusable package.
+type bindingBuilder struct {
+	apisv1alpha1.APIBinding
+}
+
+func newBindingBuilder(name string) *bindingBuilder {
+	b := new(bindingBuilder)
+	b.ObjectMeta = metav1.ObjectMeta{
+		Name: name,
+	}
+	return b
+}
+
+func (b *bindingBuilder) WithPermissionClaim(group, resource, identityHash string) *bindingBuilder {
+	if len(b.Spec.PermissionClaims) == 0 {
+		b.Spec.PermissionClaims = make([]apisv1alpha1.AcceptablePermissionClaim, 0)
+	}
+
+	pc := apisv1alpha1.AcceptablePermissionClaim{
+		PermissionClaim: apisv1alpha1.PermissionClaim{
+			GroupResource: apisv1alpha1.GroupResource{
+				Group:    group,
+				Resource: resource,
+			},
+			IdentityHash: identityHash,
+		},
```

Comment:
You ideally would record `State` and have tests that show that no warnings are displayed for both Accepted and Rejected claims

Reply:
The tests should now reflect this.